public/app/features/plugins/plugin_loader.ts

```diff
@@ -29,6 +29,17 @@ const systemJSPrototype: SystemJSWithLoaderHooks = SystemJS.constructor.prototyp
 // the content of the plugin code at runtime which can only be done with fetch/eval.
 systemJSPrototype.shouldFetch = () => true;
 
+const originalImport = systemJSPrototype.import;
+
```

Comment:
nit: a comment explaining why we need this might be useful for the future

```suggestion
// Hook Systemjs import to support plugins that only have a default export.
```